android/build.gradle

```diff
@@ -8,15 +8,16 @@ buildscript {
     ext.assertj_version = '3.13.2'
     ext.mockk_version = '1.10.0'
     ext.gradle_maven_publish = '0.11.1'
-    ext.purchases_version = '4.0.5'
+    // TODO: uncomment the following line once purchases-android update is released
+    // ext.purchases_version = '4.0.5'
 
     repositories {
         google()
         jcenter()
 
     }
     dependencies {
-        classpath "com.android.tools.build:gradle:$android_tools"
+        classpath "com.android.tools.build:gradle:${android_tools}"
```

Comment:
just curious why this one needs curly braces?

Reply:
I actually don't think it does - this was left behind after changing a lot of stuff back and forth, so I don't even recall how it got there. I'll revert this line